casper/src/main/resources/ShardConf.rho

```diff
@@ -0,0 +1,49 @@
+/*
+ The table below describes the required computations and their dependencies
+
+ No. | Dependency | Computation method | Result
+ ----+------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------
+ 1.  |            | given              | sk = a9585a0687761139ab3587a4938fb5ab9fcba675c79fefba889859674046d4a5
```

Comment:
@tgrospic @zsluedem could you guide me on how to get al this `???` so I can deploy contract to sandbox and test? I know we are going to revisit logic of blessed contracts, but I would like to test now.

Reply:
This data is generated with this tool. First unforgeable name is generated from public key + timestamp. Then as a tuple with with nonce it's signed.
https://rchain.atlassian.net/wiki/spaces/CORE/pages/660865046/Inserting+contracts+in+the+registry

I created example in JavaScript.
https://github.com/tgrospic/rnode-client-js/blob/964074d/src/nodejs/client-insert-signed.js